libraries-bom/pom.xml

```diff
@@ -52,15 +52,15 @@
     <google.cloud.core.version>2.7.1</google.cloud.core.version>
     <io.grpc.version>1.46.0</io.grpc.version>
     <http.version>1.41.8</http.version>
-    <protobuf.version>3.21.0</protobuf.version>
+    <protobuf.version>3.20.1</protobuf.version>
```

Comment:
Memo: this is from https://github.com/googleapis/java-shared-dependencies/blob/v2.12.0/first-party-dependencies/pom.xml#L62